src/partition-local-blocks.js

```diff
@@ -0,0 +1,28 @@
+// @flow
```

Comment:
instead of a separate file, i think it might make more sense to just add an option to the original function to look for local.